### PR TITLE
KONVENS-2021 Ingestion

### PR DESCRIPTION
**Pull request by xinru1414**

### CODE REVIEW

**Comment by xinru1414** [comment]
@mjpost the title of the proceeding is `Proceedings of the 17th Conference on Natural Language Processing`. Is that correct? Shouldn't it be something reflect `KONVENS`?

File: data/yaml/venues.yaml [hunk 1]
```diff
@@ -486,6 +486,9 @@ knlp:
   acronym: knlp
   name: 'Knowledgeable NLP: Workshop on Integrating Structured Knowledge and Neural
     Networks for NLP'
+konvens:
+  acronym: KONVENS
+  name: Conference on Natural Language Processing
```
Comment (mjpost) [inline data/yaml/venues.yaml:491]: Can you add a `url: https://konvens.org/site/` field?

**Review by mjpost — APPROVED** [review]

**Comment by mjpost** [comment]
I think that’s it’s English name. Titles and even venue names don’t necessarily have to be distinct, and they have the KONVENS acronym in the title, which helps.

**Comment by texttheater** [comment]
Looks good! One little thing: there doesn't seem to be a link to the full-volume PDF?

The name of the conference is indeed a little unspecific, especially in English. But yes, that is the name that has always been used. I think as long as the acronym is there it's clear enough.

**Comment by mjpost** [comment]
@xinru1414 can you check why no full-volume PDF link was created? Was this ingested from ACLPUB format or TSV? For the former there might be a bug in recognizing the book name (I recently switched it to use `cdrom/book.pdf`, I believe), and for the latter I think you have to add a command-line switch.

File: data/yaml/venues.yaml (hunk 1 above)
Comment (xinru1414) [reply data/yaml/venues.yaml:491]: done

**Pull request merged**